app/presenters/concerns/qualification_exclude_uuids.rb

```diff
@@ -0,0 +1,5 @@
+module QualificationExcludeUuids
```

Comment:
do we still need this now the default is false?

Reply:
No, we don't. I decided to add a more explicit solution but I removed it now. I also added the register api yml including the UUID fields 🚀  ♻️